frigate/http.py

```diff
@@ -825,6 +825,38 @@ def recording_clip(camera, start_ts, end_ts):
     return response
 
 
+def get_keyframe_timestamps_and_adjusted_offset(
```

Comment:
I know that we already have ffmpeg logic in the http module (which I'm hoping to clean up in the future), but seeing as this is a complete / isolated function, this might fit better inside of the `util` module. What do you think?

Reply:
Sure, I can move it there.